includes/functions.inc.php

```diff
@@ -2810,6 +2810,18 @@ function getAvatar($user_id) {
 }
 
 /**
+ * check, if HTTPS was used for the current request
+ *
+ * @return boolean
+ */
+function getURLProtocol() {
```

Comment:
The name of the function should be named "isHTTPSProtocol()" or something like that.